test/gtest/aca_test_arp.cpp

```diff
@@ -446,6 +446,11 @@ TEST(arp_request_test_cases, DISABLED_l2_arp_test_CHILD)
 
   aca_test_reset_environment();
 
+// monitor br-tun for arp request message
```

Comment:
please update the wiki page to add instruction on how to execute this DISABLED_l2_arp_test_CHILD.

Reply:
This test may be covered by ovs_l2 test and not needed.